PopPUNK/__main__.py

```diff
@@ -501,6 +520,34 @@ def main():
 
         nx.write_gpickle(genomeNetwork, args.output + "/" + os.path.basename(args.output) + '_graph.gpickle')
 
+    #******************************#
+    #*                            *#
+    #* within-strain analysis     *#
+    #*                            *#
+    #******************************#
+    sys.stderr.write("Checking options here\n\n")
```

Comment:
Remove

Reply:
Gone in effb8909e329c8ac57e1bc9aea09be1088f7021d.